.github/workflows/tests.yml

```diff
@@ -18,14 +18,14 @@ jobs:
         winterReleaseDir: ['develop']
         include:
           - phpVersion: '7.2'
-            winterRelease: 'v1.1.9'
-            winterReleaseDir: '1.1.9'
+            winterRelease: 'v1.2.1'
+            winterReleaseDir: '1.2.1'
           - phpVersion: '7.3'
-            winterRelease: 'v1.1.9'
-            winterReleaseDir: '1.1.9'
+            winterRelease: 'v1.2.1'
+            winterReleaseDir: '1.2.1'
           - phpVersion: '7.4'
-            winterRelease: 'v1.1.9'
-            winterReleaseDir: '1.1.9'
+            winterRelease: 'v1.2.1'
```

Comment:
Winter v1.2 doesn't run on PHP < 8?

Reply:
Yeah, but < 1.2 uses the old testing system so can't import `system/tests/bootstrap/app.php`, not sure what to do

Reply:
@jaxwilko see https://github.com/wintercms/wn-translate-plugin/commit/2e0049e8f2c10daa90bd7e4c73a857eb46dff013